.github/workflows/build_and_push.yml

```diff
@@ -149,29 +166,15 @@ jobs:
         kubernetes: [v1.27.2]
     steps:
       - name: Set up Go 1.19
-        uses: actions/setup-go@v4
+        uses: actions/setup-go@v5
```

Comment:
The integration test has to run before the provisioner-localpv job. It's look tidier if it was listed above the provisioner-localpv job in this yaml.